redash/models.py

```diff
@@ -553,6 +553,10 @@ def update_group_assignments(self, group_names):
     def has_access(self, obj, access_type):
         return AccessPermission.exists(obj, access_type, grantee=self)
 
+    def get_id(self):
+        identity = hashlib.md5("{0},{1}".format(self.email, self.password_hash)).hexdigest()
```

Comment:
I wonder if in some way we're revealing something about the password in this way? Maybe worth salting this? Maybe using the API key?

Reply:
Will salting really help here? I mean, the `password_hash` already acts as a salt for the email address. I don't mind appending the api key here, but I'm curious if it really adds anything.

Reply:
You're right. 👍